repose-aggregator/tests/functional-test-framework/src/main/groovy/org/openrepose/framework/test/server/CustomizableSocketServerConnector.groovy

```diff
@@ -18,7 +18,7 @@
  * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
  */
 
-package framework.server
+package org.openrepose.framework.test.server
```

Comment:
It's something @Mario-Lopez added to get the behavior he needed while still using Deproxy.